src/test/resources/pages/studentFeedbackSubmitPagePartiallyFilled.html

```diff
@@ -583,10 +583,10 @@
               <label for="input">
                 To:
               </label>
-              <select class="participantSelect middlealign newResponse form-control" name="responserecipient-6-0" style="display:none;max-width:125px">
-                <option selected="" value="">
+              <select class="participantSelect middlealign form-control" name="responserecipient-6-0" style="display:none;max-width:125px">
+                <option value="">
                 </option>
-                <option value="SFSubmitUiT.alice.b@gmail.tmt">
+                <option selected="" value="SFSubmitUiT.alice.b@gmail.tmt">
```

Comment:
Originally, at this point in the test, this question wouldn't have been answered. Hence it would not have an existing response, and the first option would have the selected attribute(line 183 of `FeedbackSubmissionEditPageData`). 

However due to line 161-173 of `StudentFeedbackSubmitPageUiTest`, which adds a response for question 6 and question 8, this question (qn 6) will already have been answered at this point. 

Hence, line 186 of `FeedbackSubmissionEditPageData` will result in `SFSubmitUiT.alice.b@gmail.tmt` having the selected attribute.